posthog/hogql_queries/error_tracking_query_runner.py

```diff
@@ -51,11 +51,35 @@ def select(self):
             ast.Alias(alias="first_seen", expr=ast.Call(name="min", args=[ast.Field(chain=["timestamp"])])),
             ast.Alias(
                 alias="description",
-                expr=ast.Call(name="any", args=[ast.Field(chain=["properties", "$exception_message"])]),
+                expr=ast.Call(
+                    name="nullIf",
+                    args=[
+                        ast.Call(
+                            name="coalesce",
+                            args=[
```

Comment:
The `coalesce` can be removed when we're happy not supporting the old properties any more. I think we should do this sooner rather than later